packages/components/src/Layout/Semantics/Layout.tsx

```diff
@@ -37,6 +37,12 @@ export interface LayoutProps
    * @default false
    */
   fixed?: boolean
+  /**
+   * scroll patterns for different areas inside the layout:
+   *  independently | x number of areas together | all together
```

Comment:
this seems to describe three different options, when in reality it's just a boolean. Maybe just lose that middle one

```suggestion
   *  independently | all together
```